scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -511,7 +513,7 @@ class Framework extends SbtFramework {
           presentReminderWithFullStackTraces,
           presentReminderWithoutCanceledTests,
           execService
-        )
+        ).asInstanceOf[Array[Task | Null]]
```

Comment:
**Overridden java array field**
A Scala field overrides some Java array field. We need to use `Array[T | Null]` instead of `Array[T]` for the overriding to work.